### PR TITLE
Updating readme

### PR DESCRIPTION
**Pull request by andrewpaliga**
Added gateways we have added + updating universal text

@girasquid @ivanfer

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -55,6 +55,7 @@ Or, if you're using Bundler, just add the following to your Gemfile:
 * [ePay](http://www.epay.dk/epay-payment-solutions/)
 * [First Data](https://firstdata.zendesk.com/entries/407522-first-data-global-gateway-e4sm-payment-pages-integration-manual)
 * [HiTRUST](http://www.hitrust.com.hk/)
+* [MOLPay](http://www.molpay.com/v2/)
```
Comment (girasquid) [inline README.md]: No country?

File: README.md [hunk 2]
```diff
@@ -63,6 +64,7 @@ Or, if you're using Bundler, just add the following to your Gemfile:
 * [PayDollar](http://www.paydollar.com)
 * [Paysbuy](https://www.paysbuy.com/) - TH
 * [Platron](https://www.platron.ru/) - RU
+* [Realex](http://www.realexpayments.com)
```
Comment (girasquid) [inline README.md]: Same question here

**Comment by girasquid** [comment]
Two questions, otherwise :+1:

File: README.md (hunk 1 above)
Comment (andrewpaliga) [inline README.md]: Country support is not stipulated on the offsite_payments level, it's only in Shopify where were restrict them so doesn't really belong in the repo.

Comment (girasquid) [inline README.md]: Other entries here have their countries - for someone reading this README, is it helpful to know what countries a gateway is for? If it's not, don't bother.

**Pull request merged**